python/ray/autoscaler/sdk/__init__.py

```diff
@@ -0,0 +1,12 @@
+from ray.autoscaler.sdk.sdk import (
+    create_or_update_cluster, teardown_cluster, run_on_cluster, rsync,
+    get_head_node_ip, get_worker_node_ips, request_resources,
+    configure_logging, bootstrap_config, fillout_defaults,
+    register_callback_handler, get_docker_host_mount_location)
+
+__all__ = [
+    "create_or_update_cluster", "teardown_cluster", "run_on_cluster", "rsync",
+    "get_head_node_ip", "get_worker_node_ips", "request_resources",
+    "configure_logging", "bootstrap_config", "fillout_defaults",
+    "register_callback_handler", "get_docker_host_mount_location"
+]
```

Comment:
Thanks for fixing this.

Asking because I don't understand Python's import system (or ray's Python module structure) -- why is the maneuver required?

Also, why in current master is `ray.autoscaler.command_runner` ok but `ray.autoscaler.sdk` not?

Reply:
> Asking because I don't understand Python's import system (or ray's Python module structure) -- why is the maneuver required?

I believe this particular maneuver is needed so that things like `help(ray.autoscaler.sdk)` and `dir(...)` work as expected. 

> Also, why in current master is ray.autoscaler.command_runner ok but ray.autoscaler.sdk not?

I don't think `ray.autoscaler.command_runner` references ray core modules in any meaningful way so it avoids circular dependencies 🤷‍♀️. I don't think I've fixed all the circular dependencies in these import paths, just enough to make this work.

Reply:
k